### PR TITLE
Fixes #2386

### LINKED ISSUE #2386: Transient failure on DruidCoordinatorTest testCoordinatorRun
testCoordinatorRun(io.druid.server.coordinator.DruidCoordinatorTest)  Time elapsed: 4.908 sec  <<< ERROR!
java.lang.Exception: test timed out after 1500 milliseconds
	at java.lang.Thread.sleep(Native Method)
	at io.druid.server.coordinator.DruidCoordinatorTest.testCoordinatorRun(DruidCoordinatorTest.java:361)

### PR DESCRIPTION
**Pull request by pjain1**
increase test time out and general clean up

### CODE REVIEW

**Comment by pjain1** [comment]
@fjy does this look good ?

File: server/src/test/java/io/druid/server/coordinator/DruidCoordinatorTest.java [hunk 1]
```diff
@@ -262,19 +262,24 @@ public String getInventoryPath()
     EasyMock.verify(metadataRuleManager);
   }
 
-  @Test(timeout = 1500L)
+  @Test(timeout = 4000L)
```
Comment (fjy) [inline server/src/test/java/io/druid/server/coordinator/DruidCoordinatorTest.java]: can we just increase this to 60_000L? it'll match the other tests

**Comment by fjy** [comment]
:+1: after travis

File: server/src/test/java/io/druid/server/coordinator/DruidCoordinatorTest.java (hunk 1 above)
Comment (pjain1) [inline server/src/test/java/io/druid/server/coordinator/DruidCoordinatorTest.java]: done

**Comment by pjain1** [comment]
Transient test failure - 
```
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 31.385 sec <<< FAILURE! - in io.druid.curator.inventory.CuratorInventoryManagerTest
testSanity(io.druid.curator.inventory.CuratorInventoryManagerTest)  Time elapsed: 31.385 sec  <<< ERROR!
org.apache.zookeeper.KeeperException$NodeExistsException: KeeperErrorCode = NodeExists for /container/billy
	at org.apache.zookeeper.KeeperException.create(KeeperException.java:119)
	at org.apache.zookeeper.KeeperException.create(KeeperException.java:51)
	at org.apache.zookeeper.ZooKeeper.create(ZooKeeper.java:783)
	at org.apache.curator.framework.imps.CreateBuilderImpl$11.call(CreateBuilderImpl.java:720)
	at org.apache.curator.framework.imps.CreateBuilderImpl$11.call(CreateBuilderImpl.java:703)
	at org.apache.curator.RetryLoop.callWithRetry(RetryLoop.java:107)
	at org.apache.curator.framework.imps.CreateBuilderImpl.pathInForeground(CreateBuilderImpl.java:699)
	at org.apache.curator.framework.imps.CreateBuilderImpl.protectedPathInForeground(CreateBuilderImpl.java:477)
	at org.apache.curator.framework.imps.CreateBuilderImpl.forPath(CreateBuilderImpl.java:467)
	at org.apache.curator.framework.imps.CreateBuilderImpl.forPath(CreateBuilderImpl.java:44)
	at io.druid.curator.inventory.CuratorInventoryManagerTest.testSanity(CuratorInventoryManagerTest.java:77)
```

**Comment by fjy** [comment]
@pjain1 please create an issue for that transient test failure

**Comment by pjain1** [comment]
@fjy done #2390

**Pull request merged**